### ISSUE TITLE
Hyperparameter model configuration tool

### ISSUE THREAD

**Issue by henrylao** [henrylao/mlacc#8]
Tuning hyper parameters can be a pain and a tool for editing the conf or creating a new experiement conf
is like to be of use. Ability to choose from a large selection of existing framework optimizers and trainers 
as well as loss functions, etc would be hte first major step

Integration with custom tooling such as internal libraries and functions would follow afterwards

The ability to very quickly and easily edit local experiement config files with tracing ﻿will be key 
for both traceability, reporudceability and orientation of where one is in reseraech
